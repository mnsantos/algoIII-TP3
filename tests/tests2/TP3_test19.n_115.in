115 2066
4 10
1 8
1 2
6 10
1 7
1 3
3 8
8 10
4 8
1 4
1 16
4 7
5 8
1 9
8 12
1 5
2 16
1 10
5 7
3 4
9 12
1 14
2 10
3 17
1 18
1 6
1 12
2 11
8 13
2 15
3 6
112 114
111 113
110 115
110 114
110 111
109 114
108 112
108 110
108 109
107 113
107 111
106 114
106 113
105 112
105 109
104 115
104 114
104 110
104 108
104 106
103 113
103 112
103 111
103 108
103 104
102 115
102 106
102 105
102 104
101 113
101 105
100 115
100 113
100 108
100 104
100 102
100 101
99 115
99 114
99 113
99 112
99 110
99 109
99 103
98 108
98 103
97 114
97 103
97 100
97 98
96 109
96 108
96 98
95 113
95 112
95 108
95 103
95 101
95 96
94 108
94 106
94 102
94 101
94 99
94 97
93 108
93 103
93 100
92 112
92 108
92 107
92 106
92 105
92 104
92 99
91 115
91 114
91 112
91 109
91 105
91 103
91 101
91 100
91 99
91 98
91 92
90 112
90 109
90 108
90 107
90 105
90 101
90 97
90 95
90 93
89 111
89 96
89 95
89 90
88 115
88 112
88 100
88 98
88 93
88 90
87 114
87 113
87 112
87 110
87 107
87 104
87 103
87 98
87 93
86 115
86 110
86 107
86 105
86 100
86 87
85 114
85 112
85 111
85 109
85 108
85 101
85 99
85 97
85 95
85 87
85 86
84 114
84 111
84 107
84 106
84 104
84 96
84 93
84 90
84 88
84 87
84 85
83 115
83 111
83 108
83 107
83 104
83 102
83 100
83 95
83 88
83 86
83 84
82 115
82 114
82 113
82 111
82 108
82 105
82 103
82 100
82 99
82 96
82 95
82 94
82 88
82 86
82 83
81 115
81 114
81 103
81 92
81 91
81 86
81 83
80 112
80 111
80 110
80 107
80 106
80 102
80 101
80 100
80 98
80 95
80 92
80 90
80 86
79 113
79 109
79 106
79 102
79 101
79 96
79 95
79 94
79 92
79 90
79 87
78 115
78 114
78 109
78 107
78 102
78 100
78 99
78 98
78 97
78 93
78 81
78 79
77 113
77 111
77 102
77 97
77 96
77 94
77 88
77 84
77 82
77 81
77 80
77 78
76 112
76 107
76 105
76 104
76 103
76 100
76 98
76 96
76 95
76 94
76 93
76 92
76 89
76 84
76 83
76 81
76 79
76 78
75 113
75 110
75 107
75 104
75 102
75 100
75 99
75 96
75 95
75 84
75 81
75 79
75 78
75 76
74 115
74 111
74 110
74 106
74 104
74 100
74 91
74 90
74 89
74 88
74 84
74 81
74 75
73 114
73 112
73 107
73 104
73 101
73 100
73 99
73 96
73 95
73 91
73 89
73 88
73 87
73 86
73 85
73 74
72 106
72 103
72 101
72 97
72 96
72 92
72 90
72 87
72 83
72 81
72 79
72 77
72 76
71 112
71 110
71 108
71 106
71 105
71 104
71 96
71 93
71 92
71 89
71 84
71 80
70 110
70 108
70 94
70 92
70 91
70 86
70 84
70 83
70 82
70 81
69 115
69 98
69 93
69 86
69 84
69 83
69 80
68 112
68 109
68 101
68 97
68 89
68 83
68 81
68 76
68 75
68 70
68 69
67 113
67 112
67 111
67 106
67 104
67 100
67 96
67 94
67 92
67 90
67 88
67 87
67 85
67 81
67 80
67 71
66 112
66 101
66 95
66 94
66 92
66 91
66 83
66 81
66 80
66 79
66 76
66 75
66 70
66 67
65 114
65 112
65 110
65 109
65 108
65 105
65 104
65 99
65 97
65 90
65 88
65 87
65 83
65 81
65 78
65 76
65 75
65 74
65 72
65 70
64 111
64 109
64 105
64 103
64 102
64 100
64 99
64 98
64 96
64 92
64 90
64 89
64 78
64 75
64 72
64 71
64 67
63 115
63 107
63 105
63 104
63 103
63 99
63 97
63 92
63 89
63 87
63 86
63 85
63 83
63 76
63 75
63 74
63 72
63 70
63 69
63 66
63 64
62 114
62 112
62 109
62 105
62 99
62 98
62 94
62 92
62 88
62 78
62 73
62 72
62 70
62 68
62 66
61 109
61 107
61 100
61 97
61 92
61 91
61 90
61 82
61 80
61 74
61 73
61 71
61 70
60 115
60 112
60 107
60 102
60 97
60 96
60 95
60 94
60 92
60 91
60 90
60 89
60 84
60 83
60 77
60 74
60 72
60 69
60 61
59 115
59 114
59 113
59 112
59 110
59 109
59 105
59 102
59 98
59 96
59 95
59 90
59 85
59 82
59 81
59 79
59 78
59 77
59 69
59 64
58 115
58 114
58 110
58 107
58 105
58 102
58 100
58 98
58 96
58 92
58 91
58 89
58 81
58 78
58 73
58 68
58 67
58 66
58 65
58 60
57 102
57 101
57 94
57 93
57 89
57 88
57 86
57 84
57 80
57 79
57 78
57 74
57 71
57 70
57 67
57 66
57 64
22 69
22 64
22 60
22 57
22 46
22 44
22 43
22 41
22 39
22 32
22 30
22 28
22 23
21 114
21 112
21 110
21 109
21 108
21 106
21 99
21 98
21 93
21 92
21 90
21 88
21 86
21 85
21 81
21 79
21 63
21 62
21 59
21 57
21 51
21 50
21 49
21 48
21 47
21 42
21 41
21 40
21 35
21 34
21 29
21 23
20 115
20 108
20 94
20 90
20 88
20 86
20 82
20 80
20 77
20 76
20 74
20 73
20 72
20 71
20 67
20 66
20 63
20 62
20 60
20 58
20 55
20 50
20 47
20 44
20 43
20 39
20 34
20 30
19 114
19 112
19 109
19 106
19 105
19 103
19 102
19 95
19 94
19 93
19 91
19 89
19 80
19 78
19 75
19 74
19 73
19 71
19 65
19 63
19 62
19 61
19 56
19 54
19 45
19 44
19 36
19 34
19 20
18 109
18 107
18 105
18 101
18 100
18 99
18 95
18 92
18 88
18 86
18 81
18 80
18 78
18 76
18 75
18 71
18 69
18 67
18 62
18 60
18 58
18 57
18 52
18 46
18 42
18 41
18 40
18 35
18 32
18 27
18 26
18 25
18 24
18 22
18 21
18 20
17 109
17 105
17 104
17 103
17 101
17 99
17 98
17 96
17 95
17 91
17 90
17 89
17 84
17 83
17 82
17 80
17 75
17 73
17 72
17 71
17 69
17 66
17 65
17 64
17 47
17 44
17 42
17 36
17 35
17 32
17 31
17 27
17 23
17 21
17 19
16 111
16 110
16 109
16 108
16 98
16 91
16 90
16 84
16 82
16 80
16 79
16 77
16 76
16 70
16 65
16 62
16 61
16 56
16 55
16 50
16 45
16 43
16 39
16 38
16 35
16 33
16 31
16 30
16 29
16 28
16 22
16 20
16 19
16 17
15 115
15 113
15 112
15 100
15 97
15 96
15 93
15 91
15 90
15 89
15 85
15 84
15 80
15 79
15 76
15 66
15 58
15 55
15 54
15 50
15 49
15 46
15 45
15 39
15 38
15 28
15 26
15 24
15 18
14 115
14 111
14 110
14 107
14 105
14 103
14 101
14 96
14 94
14 91
14 87
14 86
14 79
14 76
14 75
14 69
14 65
14 60
14 59
14 57
14 55
14 52
14 50
14 47
14 46
14 44
14 41
14 38
14 36
14 30
14 25
14 24
14 23
14 21
14 20
14 19
14 15
13 111
13 108
13 99
13 92
13 88
13 80
13 79
13 70
13 69
13 64
13 58
13 54
13 53
13 52
13 49
13 46
13 45
13 44
13 39
13 36
13 35
13 31
13 29
13 20
13 19
13 17
13 15
12 115
12 113
12 112
12 111
12 109
12 108
12 105
12 104
12 103
12 102
12 100
12 99
12 89
12 87
12 85
12 82
12 81
12 79
12 78
12 69
12 68
12 63
12 62
12 61
12 50
12 48
12 46
12 45
12 36
12 35
12 33
12 22
12 21
12 20
12 15
12 14
12 13
11 113
11 109
11 108
11 106
11 102
11 92
11 91
11 86
11 81
11 79
11 78
11 76
11 71
11 66
11 62
11 56
11 51
11 44
11 43
11 35
11 20
11 17
11 13
10 114
10 112
10 110
10 99
10 96
10 94
10 90
10 85
10 84
10 78
10 73
10 70
10 68
10 67
10 66
10 64
10 60
10 58
10 55
10 49
10 45
10 44
10 38
4 38
3 115
3 113
3 107
3 104
3 103
3 99
3 94
3 92
3 90
3 89
3 88
3 87
3 86
3 82
3 80
3 79
3 78
3 73
3 71
3 68
3 62
3 54
3 52
3 45
3 42
3 39
3 31
3 29
3 28
3 23
3 22
3 21
3 20
2 115
2 111
2 110
2 104
2 101
2 97
2 94
2 92
2 89
2 72
2 69
2 66
2 55
2 52
2 50
2 49
2 44
2 41
2 38
2 37
2 35
2 32
2 26
2 25
2 24
2 22
2 21
1 115
1 114
1 113
1 112
1 111
1 110
1 109
1 108
1 107
1 106
1 105
1 104
1 103
1 102
1 101
1 100
1 99
1 50
1 49
1 48
1 47
1 46
1 45
1 44
1 43
1 42
1 41
1 40
1 39
1 38
1 37
1 36
1 35
1 34
1 33
1 32
1 31
1 30
1 29
1 28
1 27
1 19
1 20
1 21
1 22
1 23
1 24
1 25
1 26
1 51
1 52
1 53
1 54
1 55
1 56
1 57
1 58
1 59
1 60
1 61
1 62
1 63
1 64
1 65
1 66
1 67
1 68
1 69
1 70
1 71
1 72
1 73
1 74
1 75
1 76
1 77
1 78
1 79
1 80
1 81
1 82
1 83
1 84
1 85
1 86
1 87
1 88
1 89
1 90
1 91
1 92
1 93
1 94
1 95
1 96
1 97
1 98
4 39
4 40
4 42
4 43
4 45
4 51
4 52
4 53
4 54
4 55
4 56
4 57
4 59
4 61
4 62
4 63
4 71
4 75
4 76
4 91
4 94
4 96
4 105
4 108
4 109
4 113
5 17
5 28
5 31
5 36
5 39
5 44
5 48
5 49
5 56
5 59
5 68
5 70
5 71
5 78
5 79
5 81
5 82
5 89
5 93
5 94
5 96
5 98
5 101
5 102
5 103
6 19
6 24
6 26
6 34
6 36
6 37
6 39
6 40
6 53
6 56
6 59
6 61
6 70
6 71
6 77
6 78
6 79
6 80
6 81
6 83
6 84
6 89
6 90
6 93
6 96
6 102
6 103
6 104
6 105
6 107
6 113
7 22
7 23
7 34
7 35
7 36
7 37
7 44
7 45
7 46
7 49
7 50
7 51
7 52
7 62
7 64
7 65
7 68
7 69
7 71
7 72
7 77
7 80
7 83
7 88
7 93
7 97
7 98
7 100
7 103
7 106
7 110
7 113
8 15
8 18
8 19
8 21
8 23
8 25
8 26
8 27
8 33
8 38
8 49
8 52
8 54
8 59
8 60
8 65
8 66
8 77
8 83
8 85
8 86
8 88
8 90
8 99
8 103
8 107
8 114
8 115
9 14
9 19
9 21
9 22
9 25
9 28
9 29
9 33
9 35
9 36
9 41
9 44
9 47
9 50
9 53
9 54
9 56
9 57
9 60
9 63
9 65
9 69
9 70
9 81
9 84
9 88
9 93
9 97
9 101
9 104
9 114
9 115
10 14
10 18
10 20
10 31
10 34
22 73
22 76
22 77
22 78
22 80
22 81
22 85
22 89
22 94
22 95
22 97
22 100
22 105
22 106
22 110
23 28
23 33
23 34
23 35
23 38
23 44
23 50
23 57
23 63
23 65
23 66
23 72
23 74
23 77
23 87
23 89
23 93
23 96
23 102
23 112
24 27
24 29
24 39
24 40
24 47
24 59
24 60
24 62
24 68
24 70
24 77
24 78
24 80
24 84
24 89
24 96
24 104
24 106
24 109
24 112
24 113
24 114
24 115
25 26
25 27
25 31
25 32
25 37
25 39
25 41
25 42
25 43
25 45
25 48
25 49
25 50
25 53
25 56
25 58
25 61
25 62
25 63
25 65
25 70
25 73
25 74
25 78
25 80
25 81
25 87
25 105
25 108
25 112
25 113
25 114
26 32
26 34
26 35
26 38
26 41
26 43
26 47
26 55
26 56
26 57
26 59
26 60
26 62
26 66
26 71
26 73
26 77
26 79
26 83
26 84
26 85
26 86
26 87
26 88
26 93
26 102
26 105
26 108
27 28
27 29
27 30
27 35
27 37
27 38
27 39
27 44
27 54
27 56
27 60
27 61
27 71
27 72
27 74
27 75
27 80
27 85
27 86
27 88
27 91
27 93
27 98
27 105
27 106
27 109
27 114
28 33
28 38
28 53
28 54
28 55
28 57
28 62
28 64
28 73
28 78
28 79
28 83
28 89
28 91
28 95
28 98
28 99
28 104
28 105
28 112
28 113
29 30
29 32
29 35
29 41
29 45
29 47
29 50
29 53
29 57
29 60
29 61
29 67
29 71
29 77
29 82
29 86
29 89
29 90
29 95
29 97
29 101
29 103
29 106
29 107
29 108
29 110
29 111
29 113
29 115
30 31
30 48
30 50
30 52
30 54
30 60
30 66
30 67
30 78
30 79
30 80
30 81
30 91
30 93
30 100
30 103
30 104
30 105
30 114
30 115
31 32
31 35
31 38
31 41
31 42
31 60
31 61
31 63
31 69
31 76
31 77
31 84
31 92
31 95
31 102
31 110
31 112
32 33
32 38
32 39
32 41
32 43
32 47
32 49
32 51
32 53
32 54
32 57
32 58
32 61
32 62
32 63
32 66
32 70
32 71
32 76
32 77
32 79
32 81
32 85
32 89
32 90
32 92
32 94
32 102
32 103
32 106
32 110
32 111
32 113
33 34
33 35
33 53
33 54
33 55
33 65
33 68
33 77
33 78
33 79
33 81
33 90
33 93
33 94
33 98
33 101
33 102
33 106
33 108
33 114
33 115
34 37
34 41
34 43
34 45
34 46
34 54
34 57
34 61
34 66
34 70
34 72
34 75
34 77
34 78
34 80
34 85
34 88
34 90
34 100
34 101
34 102
34 104
34 109
34 115
35 38
35 41
35 46
35 55
35 57
35 62
35 67
35 74
35 80
35 83
35 85
35 86
35 87
35 98
35 101
35 102
35 106
35 111
35 113
35 114
35 115
36 37
36 38
36 42
36 43
36 45
36 48
36 50
36 53
36 56
36 60
36 63
36 64
36 65
36 71
36 75
36 76
36 80
36 82
36 85
36 90
36 92
36 93
36 95
36 101
36 105
36 108
36 109
36 110
37 40
37 43
37 50
37 53
37 59
37 61
37 63
37 64
37 71
37 72
37 80
37 86
37 88
37 92
37 93
37 102
37 106
37 107
37 108
37 112
37 114
37 115
38 39
38 42
38 44
38 46
38 49
38 51
38 56
38 59
38 60
38 63
38 69
38 93
38 96
38 98
38 101
38 104
38 108
39 40
39 42
39 43
39 57
39 58
39 60
39 64
39 65
39 66
39 68
39 70
39 74
39 80
39 81
39 87
39 88
39 89
39 90
39 91
39 93
39 96
39 108
39 109
39 110
39 114
40 42
40 45
40 48
40 52
40 55
40 56
40 58
40 64
40 67
40 74
40 75
40 77
40 78
40 80
40 84
40 85
40 86
40 89
40 91
40 92
40 99
40 101
40 102
40 105
40 106
40 110
40 111
40 112
41 42
41 44
41 46
41 48
41 56
41 57
41 61
41 66
41 69
41 70
41 72
41 73
41 78
41 80
41 81
41 82
41 83
41 88
41 89
41 93
41 94
41 98
41 101
41 108
41 113
42 43
42 44
42 47
42 49
42 51
42 53
42 54
42 57
42 58
42 59
42 61
42 62
42 64
42 69
42 72
42 74
42 75
42 82
42 83
42 84
42 86
42 90
42 91
42 94
42 96
42 98
42 99
42 102
42 109
42 114
42 115
43 49
43 52
43 53
43 61
43 63
43 66
43 71
43 76
43 77
43 78
43 85
43 87
43 90
43 91
43 92
43 93
43 96
43 100
43 104
43 111
43 112
43 113
44 54
44 56
44 59
44 60
44 64
44 67
44 75
44 77
44 78
44 79
44 80
44 83
44 85
44 87
44 96
44 100
44 101
44 102
44 105
44 108
44 114
45 56
45 57
45 58
45 61
45 62
45 63
45 65
45 71
45 72
45 74
45 75
45 81
45 82
45 83
45 84
45 90
45 92
45 101
45 108
45 113
45 115
46 48
46 49
46 51
46 52
46 57
46 60
46 64
46 68
46 78
46 80
46 84
46 85
46 86
46 92
46 96
46 107
46 115
47 50
47 53
47 54
47 64
47 66
47 71
47 73
47 75
47 79
47 80
47 84
47 87
47 88
47 91
47 92
47 99
47 104
47 107
47 111
47 113
48 49
48 51
48 53
48 58
48 61
48 63
48 65
48 68
48 72
48 73
48 75
48 76
48 94
48 95
48 102
48 104
48 109
48 110
49 51
49 52
49 59
49 60
49 67
49 71
49 79
49 85
49 88
49 91
49 94
49 97
49 102
49 108
50 59
50 61
50 69
50 72
50 81
50 83
50 87
50 90
50 91
50 94
50 96
50 97
50 101
50 105
50 109
51 52
51 59
51 61
51 63
51 71
51 72
51 75
51 80
51 83
51 88
51 94
51 96
51 98
51 104
51 107
51 108
51 109
51 110
51 111
51 113
52 55
52 56
52 57
52 68
52 71
52 78
52 86
52 87
52 88
52 91
52 95
52 97
52 103
52 106
52 110
52 115
53 57
53 59
53 60
53 61
53 62
53 63
53 66
53 69
53 73
53 77
53 78
53 84
53 88
53 92
53 95
53 101
53 103
53 108
54 56
54 65
54 67
54 69
54 75
54 77
54 80
54 81
54 90
54 91
54 93
54 94
54 99
54 106
54 111
54 112
55 58
55 64
55 66
55 67
55 69
55 70
55 74
55 76
55 77
55 79
55 80
55 82
55 83
55 89
55 90
55 97
55 112
55 115
56 59
56 68
56 71
56 76
56 78
56 82
56 83
56 85
56 86
56 88
56 90
56 91
56 95
56 96
56 101
56 102
56 106
56 107
56 109
56 110
56 115
57 59
57 60
57 61
57 63
2 14
4 17
4 11
7 12
1 17
5 15
7 14
10 13
2 18
1 11
7 13
1 15
5 6
3 15
11 12
3 14
1 13
4 5
0
