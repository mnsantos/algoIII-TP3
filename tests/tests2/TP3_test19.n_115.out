316 7 1 18 35 41 57 80 101
